.github/workflows/win_python_matrix_test.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         Write-Host ("")
         Write-Host ("Install Pyomo...")
         Write-Host ("") 
-        python setup.py develop
+        python -W ignore setup.py develop
```

Comment:
what's that do?

Reply:
Miranda was trying to tell Python not to issue warnings.  This is killing our 2.7 Job on Windows only.  We are going to bypass Py27 testing (for a while) in GitHub actions because Windows is acting like Windows (it makes no sense at times).

Reply:
What Aaron said. So that particular flag tells Python to ignore all warning messages (which, in this case, would tell it to not print the "Python 2.7 EOL happened" user warning). But some of the pyomo commands invoke Python commands somewhere under the hood and can't be passed the `-W ignore` flag. Hence, issues.